### DISCUSSION TITLE
Having extreme difficulty getting this software to function

### DISCUSSION THREAD

**Discussion by computerfan0** [86Box/86Box#1906] [Q&A]
Whenever I try to make a machine, it demands that I set it up. I am perfectly fine with this. However, when I shut down the machine, it demands that I set it up **again**, which is absolutely not fine. In addition, clicking hard reset just closes the machine. Can anyone help me with these problems?

**Comment by jriwanek** [comment]
What are you using to start the software? running it standalone, using winbox, using 86box manager?
are you running on windows, linux, macos? what release?
Are you still experiancing the problem on the most recent release?

Your question is only marginally better than "it doesn't work!", help us to help you, provide information so we can actually help you.

**Comment by jriwanek** [comment]
Closing due to lack of user response
